src/SOS/Strike/strike.cpp

```diff
@@ -1326,6 +1326,25 @@ DECLARE_API(DumpMT)
         }
     }
 
+    ReleaseHolder<ISOSDacInterface8> sos8;
+    if (SUCCEEDED(Status = g_sos->QueryInterface(__uuidof(ISOSDacInterface8), &sos8)))
+    {
+        CLRDATA_ADDRESS assemblyLoadContext = 0;
+        Status = sos8->GetAssemblyLoadContext(TO_CDADDR(dwStartAddr), &assemblyLoadContext);
+        if (SUCCEEDED(Status))
+        {
+            const char* title = "AssemblyLoadContext:";
+            if (assemblyLoadContext != 0)
+            {
+                table.WriteRow(title, PrefixHex(assemblyLoadContext));
+            }
+            else
+            {
+                table.WriteRow(title, "Default ALC - The managed instance of this context doesn't exist yet.");
```

Comment:
Is this yet because the runtime creates an object for it at some point? Otherwise it might just be better to say:

> Default ALC - No managed context object available.

Reply:
I'm matching the wording from the `dumpalc` command. Do you prefer to up this one, both or keep them?

Reply:
No strong preference - whatever is closer to guiding people to the actual behavior. Yet sounds like there's the expectation it will exist later. I don't know of a managed representation of the default ALC hence the suggestion, but you are more familiar with them that I am.

Reply:
The default ALC is the only one that can exist but not have a managed representation. If the user wants the default ALC, we construct a managed representation on the demand.